packages/docusaurus-plugin-client-redirects/src/templates/redirectPage.template.html.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default `
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="refresh" content="0; url=<%= it.toUrl %>">
+    <link rel="canonical" href="<%= it.toUrl %>" />
```

Comment:
I wonder if it's useful to include a robots noIndex here? Not sure this is very useful but if you know...